### PR TITLE
Install and configure Tailwind CSS

### LINKED ISSUE #18: Choose CSS enhancement
It would be beneficial to use a CSS enhancer for this website, as vanilla CSS is lacking in many areas. Most options should be easy to integrate with the project, as the CRA build system already supports most of them.

A decision should be reached by multiple consensus. Some of the popular options are:

- [Tailwind](https://tailwindcss.com/)
- [Less](https://lesscss.org/)
- [SASS/SCSS](https://sass-lang.com/)

- [ ] Install preprocessor
- [ ] Update and organize all style files (if applicable)

### PR DESCRIPTION
**Pull request by ThePotatoArchivist**

### CODE REVIEW

**Comment by ThePotatoArchivist** [comment]
Will be rebased onto main after #25 is merged

**Comment by NathanDuPont** [comment]
rebased & squashed, also added a style to confirm tailwind was working.

Adding the following style to the code (available on the tailwind website):
```
<h1 className='text-fuchsia-500'>Welcome to </h1>
```

We can see that the text updates appropriately:
image

Also confirmed that pages can be switched between (both through the post link, and through the "Home" button on the navbar)

File: src/routes/articles/Post1.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const Post1 = () => (
     <main>
-        <h1>Welcome to </h1>
```
Comment (NathanDuPont) [inline src/routes/articles/Post1.tsx:3]: this was needed, don't @ me

File: src/routes/articles/Post1.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+        <h1 className='text-fuchsia-500'>Welcome to </h1>
```
Comment (NathanDuPont) [inline src/routes/articles/Post1.tsx:3]: this was needed, don't @ me

**Review by natwal7 — APPROVED** [review]
:)

**Pull request merged**